Templates/E-Commerce/Shopify/Shopify to Salesforce - Products and prices/long-description.md

```diff
@@ -0,0 +1,23 @@
+This template inserts and updates Shopify products and their corresponding prices to Salesforce.
+Using this template, you can insert your new Shopify products and prices to Salesforce and update existing products' prices.
+
+![Template](assets/Shopify_to_Salesforce_-_Products_and_prices.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Salesforce user should have access to client ID, secret and security token.
+- Price book for Shopify is initialized in Salesforce. It is labelled as "Shopify Price Book" in this template.
+
+# Implementation and Usage Notes
+
+This template uses Salesforce queries to identify a product and the price book it's connected to. All products are checked, and if the product exists in Salesforce, the price for that item will be updated in the Shopify price book with latest information from Shopify.
+Standard price book status information is retrieved and compared to Shopify price book name. If Shopify is not the standard price book, the product data is inserted into the standard price book first. After that, the Shopify price book information is retrieved for product data handling.
```

Comment:
I think in markdown you have to have an empty line inbetween to actually make it two separate paragraphs. Right now all these lines will form just 1 long paragraph